h1st/core/trust/describable.py

```diff
@@ -31,6 +31,8 @@ def describe(self, constituency=Constituency.ANY, aspect=Aspect.ANY):
                 out : Description of Model's behavior and properties
         """
         describer = Describer(self)
+        if not all(hasattr(self, attr) for attr in ["_native_model", "prepared_data"]):
```

Comment:
@timroz24 it is quite magic for "_native_model" and "prepared_data". Class Describable should have a way for sub class to explicitly inject these values.

Reply:
This is actually quite terrible. This requirement is strange, not documented anywhere, and I have to guess at the intent from reading the code. I don’t necessarily like @thangtp’s suggested solution, either. @timroz24 please back up to thinking about what you’re trying to accomplish, and find a more canonical solution. This is framework code and API, and cannot have weird stuff like this. Thanks.